README.md

```diff
@@ -179,10 +179,6 @@ CARGO_BUILD_PIPELINING=true cargo build
 
 [p]: https://internals.rust-lang.org/t/evaluating-pipelined-rustc-compilation/10199
 
-### Getting the rest of the system working
-
-To get other components ([TiDB](https://github.com/pingcap/tidb) and [PD](https://github.com/pingcap/pd)) working, we suggest you follow the [development guide](https://github.com/pingcap/docs/blob/master/dev-guide/development.md), because you need the `pd-server` at least to work alongside `tikv-server` for integration level testing.
-
```

Comment:
Removed this part since this claims TiKV as co-components of TiDB and PD. Hence the link to dev guide is not needed. @breeswish

Reply:
Added back but removed TiDB.